### DISCUSSION TITLE
Takehisa Kosugi + - 1987 for neoscore Formalization

### DISCUSSION THREAD

**Discussion by jgarte** [DigiScore/neoscore#47] [Ideas]
wdyt

![kosugi](https://user-images.githubusercontent.com/47760695/175182184-e845737c-2dc0-43f8-9764-0eb688802810.gif)

http://iberia.music.coocan.jp/score_gallery_kosugi1.htm

**Comment by ajyoon** [comment]
This would also be very straightforward to implement in Neoscore. We'd need to get permission to use it though, and IMO technically it seems so simple as to be not very useful as an example.
